Test.C

```diff
@@ -414,24 +469,27 @@ void equipmentStore(struct Player *playerPtr, struct Equipment equipmentStock[NU
     printf("|------------------------------ Equipment  Store ------------------------------|\n");
     printf("You currently have %i credits\n", playerPtr->currentCredits);
 
-    printf(" # | Equipment Name       | Cost  | Message Amount\n");
-    printf("---|----------------------|-------|---------------\n");
+    printf(" # | Equipment Name                         | Cost  | Message Amount\n");
+    printf("---|----------------------------------------|-------|---------------\n");
     for(int i=0; i<NUM_EQUIPMENT; i++){
-        printf(" %d | %20s | %5d | %5d\n", i + 1, equipmentStock[i].name, equipmentStock[i].cost, equipmentStock[i].messageAmount);
+        printf(" %d | %40s | %5d | %5d\n", i + 1, equipmentStock[i].name, equipmentStock[i].cost, equipmentStock[i].messageAmount);
```

Comment:
Reduce 40 to 38 to keep under 80 characters and everything lined up with the other tables.
```suggestion
        printf(" %d | %38s | %5d | %5d\n", i + 1, equipmentStock[i].name, equipmentStock[i].cost, equipmentStock[i].messageAmount);
```